src/game/game_state.c

```diff
@@ -180,9 +180,11 @@ int game_state_create(game_state *gs, engine_init_flags *init_flags) {
             omf_free(gs->players[i]->pilot);
             gs->players[i]->pilot = &gs->rec->pilots[i].info;
             // this function alters the palette
-            sd_pilot_set_player_color(gs->players[i]->pilot, PRIMARY, gs->rec->pilots[i].info.color_3);
-            sd_pilot_set_player_color(gs->players[i]->pilot, SECONDARY, gs->rec->pilots[i].info.color_2);
-            sd_pilot_set_player_color(gs->players[i]->pilot, TERTIARY, gs->rec->pilots[i].info.color_1);
+            if(i == 0 || gs->rec->game_mode != REC_GAMEMODE_TOURNAMENT) {
```

Comment:
Maybe comment that we always use altpals for the first player (although I believe it's possible to have a tournament player that has custom colors from the PIC file).

Reply:
Just realized: with the check I added in "Don't load altpals index 16 (default in omf tournament mode", this `if` is redundant.

Thanks for bringing my attention back to it.